Source/Core/Core/NetPlayClient.cpp

```diff
@@ -1035,6 +1056,31 @@ u8 NetPlayClient::LocalWiimoteToInGameWiimote(u8 local_pad)
 	return ingame_pad;
 }
 
+void NetPlayClient::HashRam()
+{
+	std::lock_guard<std::mutex> lk(crit_netplay_client);
+	bool isPlayer = false;
+	for (int i = 0; i < 4; ++i)
+	{
+		if (netplay_client->m_pad_map[i] == netplay_client->m_local_player->pid)
+			isPlayer = true;
+		if (netplay_client->m_wiimote_map[i] == netplay_client->m_local_player->pid)
+			isPlayer = true;
+	}
+
+	if (isPlayer)
+	{
+		bool bWii = SConfig::GetInstance().m_LocalCoreStartupParameter.bWii;
+		u64 hash = XXH64(Memory::GetPointer(bWii ? 0x90000000 : 0x80000000), bWii ? 0x4000000 : 0x2000000, 0);
```

Comment:
I don't know the details of how the FakeVMem hack interacts with PPC state; actual games won't store anywhere near 32MB of data there, but I'm not sure how it's laid out.

You don't need to hash it for Wii games, so Wii games will still hash more total memory than GC games.  (Actually, FakeVMem should probably be forced off for Wii games.)